### ISSUE TITLE
Empty options column

### ISSUE THREAD

**Issue by chtotut** [henninglive/logi-led#9]
Does the `config` example work?
I have an empty column with options in Logitech Gaming Software for `config` applet.
Version 8.98.234 x86-64

**Comment by henninglive** [comment]
No, I haven't gotten it to work. It's one of the things I wanted to look into before publishing the crate. I also ran into lots of race condition problems when testing, that's why the examples have delays everywhere.
